kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md

```diff
@@ -32,7 +33,7 @@ In the comments next to each cell, we've marked which component of the YellowPap
 
 ```k
     configuration
-        <k> #loadProgram($PGM:Bytes) ~> #precompiled?($ACCTCODE:Int, getSchedule($SCHEDULE:Int)) ~> #execute </k>
+        <k> #loadProgram($PGM:Bytes) ~> #precompiled?(Int2MInt($ACCTCODE:Int)::MInt{256}, getSchedule($SCHEDULE:Int)) ~> #execute </k>
```

Comment:
Do we need to convert this on the c++ side to an Int or is there a way to pass it directly as an MInt? Did you look into this?

Reply:
I used the pattern we were following to initialize the K-cells, but in theory, it should be possible to use our `llvm_i256` struct and use the wrapper to send it to KEVM and then unwrap it here. Right, @theo25?

Reply:
yes we could do that for all parameters that are passed by the C++. I have not tried it but in theory it should work.

Reply:
https://github.com/Pi-Squared-Inc/pi2/issues/2806